cvat-core/src/session-implementation.ts

```diff
@@ -491,7 +492,15 @@ export function implementTask(Task) {
             ...(typeof this.cloudStorageId !== 'undefined' ? { cloud_storage_id: this.cloudStorageId } : {}),
         };
 
-        const task = await serverProxy.tasks.create(taskSpec, taskDataSpec, onUpdate);
+        const { taskID, rqID } = await serverProxy.tasks.create(
+            taskSpec,
+            taskDataSpec,
+            options?.uploadStatusCallback || (() => {}),
+        );
+        await requestsManager.listen(rqID, {
+            callback: options?.requestStatusCallback,
+        });
```

Comment:
Did you decide to generate server id on client and use common endpoint?

Reply:
Yes, I think it has advantages: We have guessing logic in one place - serverProxy, so we dont need to invent client ids and then parse it to send filter request.  
If something changes on server, we can change one line on the client(but I suppose next change on the server should be returning correct id)

Reply:
Okay